src/js/structs/StateSummary.js

```diff
@@ -53,7 +61,9 @@ module.exports = class StateSummary {
   }
 
   getActiveSlaves() {
-    return _.where(this.snapshot.slaves, {active: true});
+    return this.snapshot.slaves.filter(function (slave) {
```

Comment:
It didn't make sense to me that `.active` would be anything but a boolean but it can be changed back to explicit `slave.active === true` if you think it's important?